From f54c82f4732035296e252a0a4368efa56549ba35 Mon Sep 17 00:00:00 2001
From: Giulio Benetti <giulio.benetti@micronovasrl.com>
Date: Fri, 5 Jan 2018 23:01:02 +0100
Subject: [PATCH] Enable parallel building passing variable to Makefile.

Add JOBS variable to driver's Makefile.
JOBS variable will contain the jobs number to use when calling sub-make.

Signed-off-by: Giulio Benetti <giulio.benetti@micronovasrl.com>
---
 src/devicedrv/mali/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/devicedrv/mali/Makefile b/src/devicedrv/mali/Makefile
index 66a2b28..c3790a0 100755
--- a/src/devicedrv/mali/Makefile
+++ b/src/devicedrv/mali/Makefile
@@ -194,18 +194,18 @@ EXTRA_DEFINES += -DMALI_MEM_SWAP_TRACKING=1
 endif
 
 modules: $(UMP_SYMVERS_FILE)
-	$(MAKE) ARCH=$(ARCH) -C $(KDIR) M=$(CURDIR) modules
+	$(MAKE) -j$(JOBS) ARCH=$(ARCH) -C $(KDIR) M=$(CURDIR) modules
 	@rm $(FILES_PREFIX)__malidrv_build_info.c $(FILES_PREFIX)__malidrv_build_info.o
 
 all: modules
 
 clean:
-	$(MAKE) ARCH=$(ARCH) -C $(KDIR) M=$(CURDIR) clean
+	$(MAKE) -j$(JOBS) ARCH=$(ARCH) -C $(KDIR) M=$(CURDIR) clean
 
 install: modules
-	$(MAKE) ARCH=$(ARCH) -C $(KDIR) M=$(CURDIR) INSTALL_MOD_PATH=$(INSTALL_MOD_PATH) modules_install
+	$(MAKE) -j$(JOBS) ARCH=$(ARCH) -C $(KDIR) M=$(CURDIR) INSTALL_MOD_PATH=$(INSTALL_MOD_PATH) modules_install
 
 kernelrelease:
-	$(MAKE) ARCH=$(ARCH) -C $(KDIR) kernelrelease
+	$(MAKE) -j$(JOBS) ARCH=$(ARCH) -C $(KDIR) kernelrelease
 
 export CONFIG KBUILD_EXTRA_SYMBOLS
-- 
2.7.4

